lib/dotcom/system_status/subway.ex

```diff
@@ -11,6 +11,7 @@ defmodule Dotcom.SystemStatus.Subway do
   @type status_t() :: :normal | Dotcom.SystemStatus.Alerts.service_effect_t()
 
   @type status_entry() :: %{
+          alerts: Alert.t(),
```

Comment:
Question: Should this be a list of alerts, rather than a single alert?
```suggestion
          alerts: [Alert.t()],
```

(I think [Dialyzer might be complaining about this as well](https://github.com/mbta/dotcom/actions/runs/13397606493/job/37420443978?pr=2388), hence the failing build)